src/test/resources/odf1.4/tools/copy-xslt-output-from-target-to-references.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
```

Comment:
please add this to every shell script you write to turn on "strict mode":

set -euo pipefail